x-pack/plugins/maps/public/classes/sources/es_source/es_source.ts

```diff
@@ -157,26 +160,28 @@ export class AbstractESSource extends AbstractVectorSource implements IESSource
 
   async _runEsQuery({
     registerCancelCallback,
-    requestDescription,
     requestId,
     requestName,
     searchSessionId,
     searchSource,
     executionContext,
     requestsAdapter,
+    onWarning,
   }: {
     registerCancelCallback: (callback: () => void) => void;
-    requestDescription: string;
```

Comment:
I'm slightly concerned that we may be removing useful context for the user when removing the `requestDescription`. Did you find the `requestDescription` was not helpful or inaccurate or maybe some other reason?

Reply:
I found requestDescription to be dive too deeply into implementation details. The user only cares about what is getting loaded, like layer features or join metrics. Details about how layer features are elasticsearch aggregation buckets vs elasticsearch documents is not important and should not be surfaced at a high level. Users can interrogate the request if they really want all the implementation details.